packages/block-library/src/button/edit.native.js

```diff
@@ -500,7 +499,6 @@ class ButtonEdit extends Component {
 							</ToolbarGroup>
 						</BlockControls>
 						{ this.getLinkSettings( false ) }
-						<ColorEdit { ...this.props } />
```

Comment:
Was the color edit only used for gradients in mobile?

Reply:
It was used for both color and gradients, we recently merged part of the implementation of the Global styles where we enabled the colors hook usage.